test/e2e/e2e_test.go

```diff
@@ -23,19 +23,20 @@ import (
 )
 
 const (
-	vxlanPort            = "4789" // IANA assigned VXLAN UDP port - rfc7348
-	podNetworkAnnotation = "k8s.ovn.org/pod-networks"
-	retryInterval        = 1 * time.Second  // polling interval timer
-	retryTimeout         = 40 * time.Second // polling timeout
-	ciNetworkName        = "kind"
+	vxlanPort                   = "4789" // IANA assigned VXLAN UDP port - rfc7348
+	podNetworkAnnotation        = "k8s.ovn.org/pod-networks"
+	retryInterval               = 1 * time.Second  // polling interval timer
+	retryTimeout                = 40 * time.Second // polling timeout
+	ciNetworkName               = "kind"
+	ovnK8sNamespace      string = "ovn-kubernetes" //OVN kubernetes namespace
```

Comment:
nit: since the type is dynamically inferred, you can omit the `string`

Reply:
i intentionally kept it typed, as it was define down in each test set.